skbio/stats/composition.py

```diff
@@ -608,10 +612,274 @@ def centralize(mat):
 
     """
     mat = closure(mat)
-    cen = ss.gmean(mat, axis=0)
+    cen = scipy.stats.gmean(mat, axis=0)
     return perturb_inv(mat, cen)
 
 
+@experimental(as_of="0.4.0-dev")
+def ancom(matrix, categories,
+          alpha=0.05,
+          tau=0.02,
+          theta=0.1,
+          multiple_comparisons_correction=None,
+          significance_test=scipy.stats.ttest_ind):
+    r"""
+    Calculates pairwise log ratios between all otus
```

Comment:
"otus" and "features" are being used interchangeably. Can you pick one for consistency? I recommend "features" because it's more general (you could list OTUs as an example type of feature).